### ISSUE TITLE
Feature Request -  Support React Native

### ISSUE THREAD

**Issue by gaodeng** [rbren/rss-parser#79]
Remove browser's window global variable dependency, build a umd library.

**Comment by bobby-brennan** [comment]
Happy to accept a PR to this effect!

Probably related to #53

**Comment by yamatsum** [comment]
Thank you for the wonderful package!

It was available in RN in the way described in #53.
However, the following warning will be given.

I'm looking into it, 
but I think that it is probably because a deprecated one is specified in 
responseType of XMLHttpRequest in the file generated by webpack.
Please let me know if you know how to fix it.

```
[00:37:40] Warning: The provided value 'moz-chunked-arraybuffer' is not a valid 'responseType'.
- node_modules/fbjs/lib/warning.js:33:20 in printWarning
- node_modules/fbjs/lib/warning.js:57:25 in warning
- node_modules/react-native/Libraries/Network/XMLHttpRequest.js:189:8 in set
- node_modules/rss-parser/dist/rss-parser.min.js:7:56194 in n
- node_modules/rss-parser/dist/rss-parser.min.js:7:56681 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:7:56804 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:7:111092 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:7:115176 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:7:54922 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:7:55792 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:7:105499 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:7:105118 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:7:105072 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:1:452 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:461 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:292:12 in loadModuleImplementation
* App.tsx:5:0 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:292:12 in loadModuleImplementation
- node_modules/expo/AppEntry.js:2:0 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:292:12 in loadModuleImplementation
- node_modules/metro/src/lib/polyfills/require.js:179:45 in guardedLoadModule
* null:null in global code

[00:37:41] Warning: The provided value 'ms-stream' is not a valid 'responseType'.
- node_modules/fbjs/lib/warning.js:33:20 in printWarning
- node_modules/fbjs/lib/warning.js:57:25 in warning
- node_modules/react-native/Libraries/Network/XMLHttpRequest.js:189:8 in set
- node_modules/rss-parser/dist/rss-parser.min.js:7:56194 in n
- node_modules/rss-parser/dist/rss-parser.min.js:7:56629 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:7:56804 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:7:111092 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:7:115176 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:7:54922 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:7:55792 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:7:105499 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:7:105118 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:7:105072 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:105 in e
- node_modules/rss-parser/dist/rss-parser.min.js:1:452 in <unknown>
- node_modules/rss-parser/dist/rss-parser.min.js:1:461 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:292:12 in loadModuleImplementation
* App.tsx:5:0 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:292:12 in loadModuleImplementation
- node_modules/expo/AppEntry.js:2:0 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:292:12 in loadModuleImplementation
- node_modules/metro/src/lib/polyfills/require.js:179:45 in guardedLoadModule
* null:null in global code
```
